README.md

```diff
@@ -32,8 +32,9 @@ Alpine Linux and is using Docker.
 The installation of this add-on is pretty straightforward and not different in
 comparison to installing any other Hass.io add-on.
 
-**NOTE**: Your Hass.io machine MUST HAVE a static IP for this add-on to function
+**NOTE 1**: Your Hass.io machine MUST HAVE a static IP (not DHCP reserved, but static) for this add-on to function
```

Comment:
Always ran pihole (in and outside hass.io) reserved dhcp, no issue what so ever.
Maybe its different when you enable the dhcp server? But then this note still gives the wrong impression.

Reply:
Incorrect. First off all, we are talking about inside hassio here. 

Secondly if Pihole act as dhcp or not is not a factor in this.

This not is actually correct. Not following it and using some kind of DHCP in your network to provision Pihole dns onto your clients, will eventually cause issues.

Reply:
So all my hassio test setups shouldn't work? 
Well, guess I have some magic going then!

Could you explain the "issues" at least? does Hassio use forced DNS for the docker containers of some sort? That might actually cause issues if it does!

Reply:
I think I get it now:
You are primarily concerned with the DNS adress the DHCP server gives out (the pi hole)? The rest of the static IP isn't the issue, just the fact it gives out the pihole DNS IP... Now I get you!